content/docs/setup/kubernetes/multicluster-install/zvpn/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Using Istio Gateway
+description: Connect multiple clusters using Istio Gateway to reach remote pods.
+weight: 2
+keywords: [kubernetes,multicluster,gateway]
+---
+
+Instructions for spanning an Istio mesh across multiple clusters when pods
+in each cluster can only talk to remote gateway IPs.
+
+## Prerequisites
+
+* Two or more Kubernetes clusters with **1.9 or newer**.
+
+* The ability to deploy the [Istio control plane](/docs/setup/kubernetes/quick-start/)
+on **each** Kubernetes cluster.
+
+* The gateway IP (load balancer IP) in each cluster must be accessible
+from every other cluster.
+
+* **Root CA**: To enable mTLS communication across clusters, each cluster's
+  Citadel must be configured with an intermediate CA credentials generated
+  by a shared root CA.
+
+## Overview
+
+In this mode, each cluster has an **identical** Istio control plane
+installation. Cross-cluster communication occurs over Istio Gateways
+of the respective clusters. 
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-zvpn.svg"
+    caption="Spanning a single mesh across multiple clusters using Istio Gateways"
+    >}}
+
+This guide describes steps to install a multicluster Istio topology using
```

Comment:
change multicluster Istio tpology using the manifests -> gateway-based mesh of connected clusters using the manifests